### PR TITLE
yinghuacd.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/140122

### Add your comment and screenshots

<details><summary>Screenshots</summary>

Screenshot from 2023-01-17 11-20-58
Screenshot from 2023-01-17 11-20-46

</details>

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@ allets.ru,annahelp.ru,newslenta.com,kogdavyydet.com,u74.ru,wifika.ru,mater.com.u
...
 !
+||push.zhanzhang.baidu.com^
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:66]: Shouldn't it be added to social_trackers? @Alex-302 please take a look.

Comment (Alex-302) [reply SocialFilter/sections/specific.txt:66]: As I see, this is analytics. Please look at script's content.

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: I had seen it and thougth the same, but wasn't sure for 100% that's why I mentioned you. Ok.

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: @piquark6046 please move it to AdGuard Tracking Protection, it looks like a tracker and I agree with it https://github.com/AdguardTeam/AdguardFilters/pull/140316#discussion_r1072192956.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**